Parse/src/main/java/com/parse/Parse.java

```diff
@@ -158,72 +152,73 @@ public Builder server(String server) {
       }
 
       /**
-       * Add a {@link ParseNetworkInterceptor}.
-       *
-       * @param interceptor The interceptor to add.
-       * @return The same builder, for easy chaining.
-       */
-      public Builder addNetworkInterceptor(ParseNetworkInterceptor interceptor) {
-        if (interceptors == null) {
-          interceptors = new ArrayList<>();
-        }
-        interceptors.add(interceptor);
-        return this;
-      }
-
-      /**
        * Enable pinning in your application. This must be called before your application can use
        * pinning.
+       *
        * @return The same builder, for easy chaining.
        */
       public Builder enableLocalDataStore() {
         localDataStoreEnabled = true;
         return this;
       }
 
-      /* package for tests */ Builder setNetworkInterceptors(Collection<ParseNetworkInterceptor> interceptors) {
```

Comment:
Yeah, we are just going to have to remove support for that library and instead point users to using Stetho on their OkHttpClient Builder that they create for Parse http://facebook.github.io/stetho/

Somewhat unfortunate, but I think it is the best thing moving forward